### PR TITLE
NETOBSERV-465: Kafka metrics 2

### PR DESCRIPTION
**Pull request by OlivierCazade**
Added more metrics to the kafka ingester and added timeout mecanics to the kafka tests.

### CODE REVIEW

File: pkg/pipeline/ingest/ingest_kafka.go [hunk 1]
```diff
@@ -74,14 +76,61 @@ func (ingestK *ingestKafka) kafkaListener() {
...
 
 }
 
+func processRecordDelay(record config.GenericMap) {
+	TimeFlowEndInterface, ok := record["TimeFlowEnd"]
+	if !ok {
+		flowErrors.With(prometheus.Labels{"router": "", "error": "No TimeFlowEnd found"}).Inc()
+		return
+	} else {
+		fmt.Println(TimeFlowEndInterface)
```
Comment (KalmanMeth) [inline pkg/pipeline/ingest/ingest_kafka.go:101]: Do you really want to print to stdout for each end of flow? or is this only for debug purposes?

Comment (OlivierCazade) [reply pkg/pipeline/ingest/ingest_kafka.go:101]: Definitly not, this is a debug trace that I forgot to remove.
Thanks

File: pkg/test/utils.go [hunk 2]
```diff
@@ -163,3 +165,14 @@ func DeserializeJSONToMap(t *testing.T, in string) config.GenericMap {
 	require.NoError(t, err)
 	return m
 }
+
+func WaitFromChannel(in chan []config.GenericMap, timeout int64) ([]config.GenericMap, error) {
+	duration := time.Duration(timeout) * time.Millisecond
```
Comment (ronensc) [inline pkg/test/utils.go:170]: Won't it be more idiomatic to change the type of `timeout` from `int64` to `time.Duration`?

Then, in the invocation side, one can argue that it is more apparent what the unit of time is:
```go
test.WaitFromChannel(ingestOutput, 2 * time.Second)
```

**Review by ronensc — APPROVED** [review]

File: pkg/test/utils.go (hunk 2 above)
Comment (OlivierCazade) [reply pkg/test/utils.go]: Good point thanks!
I changed it.

**Review by jpinsonneau — APPROVED** [review]

**Pull request merged**